[
  {
    "type": "value_error",
    "loc": [
      "contributor",
      0,
      "Person"
    ],
    "msg": "Value error, Contact person must have an email address.",
    "input": {
      "name": "Steinmetz, Nicholas",
      "roleName": [
        "dcite:Author",
        "dcite:ContactPerson"
      ],
      "schemaKey": "Person",
      "identifier": "0000-0001-7029-2908",
      "includeInCitation": true
    },
    "ctx": {
      "error": "Contact person must have an email address."
    },
    "url": "https://errors.pydantic.dev/2.10/v/value_error"
  }
]